exe-unit/components/gsb-http-proxy/src/gsb_to_http.rs

```diff
@@ -53,9 +53,9 @@ impl GsbToHttpProxy {
 
     pub fn bind(&mut self, gsb_path: &str) -> Handle {
         let this = self.clone();
-        bus::bind(gsb_path, move |message: GsbHttpCallMessage| {
+        bus::bind_with_caller(gsb_path, move |caller, message: GsbHttpCallMessage| {
```

Comment:
Do I understand correctly that it aims to resolve https://github.com/golemfactory/ya-runtime-ai/issues/105?

If so, how does it solve the problem if we don't make use of that parameter in `pass()` apart from logging? Or do we only want to log the messages for now and fix it after collecting more data?

Reply:
Or do we solve the problem by switching `authorize_activity_executor()` to `authorize_activity_initiator()` and log to ensure that there are no problems with it after the change?

Reply:
Yes it's solves the problem with spam that reports incorrectly that something is wrong when it isn't.
I don't want to block possibility of using other yagna as proxy, so the call is not rejected even if it is from incorrect identity